src/config/mongoose_config_parser_toml.erl

```diff
@@ -111,7 +112,34 @@ process_general(<<"max_fsm_queue">>, V) ->
 process_general(<<"http_server_name">>, V) ->
     [#local_config{key = cowboy_server_name, value = b2l(V)}];
 process_general(<<"rdbms_server_type">>, V) ->
-    [#local_config{key = rdbms_server_type, value = b2a(V)}].
+    [#local_config{key = rdbms_server_type, value = b2a(V)}];
+process_general(<<"override_local">>, V) ->
+    [#local_config{key = override_local, value = V}];
+process_general(<<"override_global">>, V) ->
+    [#local_config{key = override_global, value = V}];
```

Comment:
It might have been the `#config` record instead. Does not really matter since it is not in the final config anyway, these `override` options are stored separately in `#state` record. In TOML we don't have a global `State` while parsing so this is a workaround to later remove them from the config and add them to the appropriate `#state` field.

Reply:
Maybe we could introduce another temporary config type in this case to avoid the confusion? It could be even `{override, local | global | acls}` that would be there only if the value is `true` - otherwise return empty list.